src/schemas/validation/schema.yaml

```
@@ -230,6 +230,24 @@ $defs:
         type: array
         items:
           $ref: '#/$defs/parameter-or-reference'
+      additionalOperations:
+        type: object
+        additionalProperties:
+          $ref: '#/$defs/operation'
+        propertyNames:
+          $comment: RFC9110 restricts methods to "1*tchar" in ABNF
+          pattern: "^[a-zA-Z0-9!#$%&'*+.-^`|~]+$"
```

The character class will currently want to interpret `.-^` as a character range - the `-` should be at the end or beginning. `\`-escaping is also an option but I think putting it at an end is more universal.

---

@notEthan I thought I'd had it at the end... yeah this needs fixing.
```suggestion
          pattern: "^[a-zA-Z0-9!#$%&'*+.^`|~]+$-"
```

---

```suggestion
          pattern: "^[a-zA-Z0-9!#$%&'*+.^`|~-]+$"
```